### ISSUE TITLE
Example not working with Ubuntu 22.04 LTS

### ISSUE THREAD

**Issue by reisenmachtfreude** [karask/python-bitcoin-utils#34]
Hi there,

I've problems with running the example code in the file: https://github.com/karask/python-bitcoin-utils/blob/b1bc4ebdade5521f3d60b7acea1b99ac94d18635/examples/keys_addresses.py
```
     hashripemd160 = hashlib.new('ripemd160')
  File "/usr/lib/python3.10/hashlib.py", line 166, in __hash_new
    return __get_builtin_constructor(name)(data)
  File "/usr/lib/python3.10/hashlib.py", line 123, in __get_builtin_constructor
    raise ValueError('unsupported hash type ' + name)
ValueError: unsupported hash type ripemd160
```
What should I do?

I'm using Ubuntu 22.04 LTS

**Comment by karask** [comment]
Hi! I was very recently made aware of this (new version of the library).

A workaround can be found here:
https://stackoverflow.com/questions/72409563/unsupported-hash-type-ripemd160-with-hashlib-in-python

However, I will try find some time to fix this properly asap.

**Comment by karask** [comment]
Fixed in v0.5.8 in accordance to https://github.com/bitcoin/bitcoin/issues/23710
